protos/base_node/index.js

```diff
@@ -3,6 +3,17 @@ const fetch = require('node-fetch')
 const md5 = require('md5')
 const fs = require('fs')
 const path = require('path')
+
+const defaultHeightOrBlockGroupRequest = {
+  from_tip: 100,
+  start_height: 100,
+  end_height: 100
```

Comment:
Approximately the last hour in blocks from the tip.
```suggestion
  from_tip: 60,
  start_height: 0,
  end_height: 0
```

Reply:
```suggestion
  from_tip: 30,
  start_height: 0,
  end_height: 0
```